src/ucp/api/ucp_compat.h

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
pls change this from a doxygen comment block to a regular C-style comment block (/**  ->  /*)